src/DiagnosticSourceListener/DiagnosticSourceListener.csproj

```diff
@@ -52,6 +52,7 @@
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
     <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="4.5.0" />
+    <PackageReference Include="System.Net.Http" Version="4.3.4" />
```

Comment:
this is only applicable to netstandard (<2.0) targets and is bad for netfx targets
could you please make sure for all projects we ally it to  netstandard1.3 and 1.6 only?

Reply:
Thank you. I will do this